app/scripts/migrations/135.ts

```diff
@@ -0,0 +1,187 @@
+import { hasProperty, isObject } from '@metamask/utils';
+import { cloneDeep } from 'lodash';
+
+type VersionedData = {
+  meta: { version: number };
+  data: Record<string, unknown>;
+};
+
+export const version = 135;
+
+/**
+ * Removes tokens with `decimals === null` from `allTokens`, `allIgnoredTokens`, `allDetectedTokens`, `tokens`, and `detectedTokens`.
```

Comment:
`allIgnoredTokens` is no longer migrated.

```suggestion
 * Removes tokens with `decimals === null` from `allTokens`, `allDetectedTokens`, `tokens`, and `detectedTokens`.
```